msteams-platform/samples/virtual-table-entity-references.md

```diff
@@ -0,0 +1,292 @@
+|`m365_originalstart`|DateTimeType|Represents the start time of an event when it's initially created as an occurrence or exception in a recurring series. This property isn't returned for events that are single instances. Its date and time information is expressed in ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014 is 2014-01-01T00:00:00Z|
+|`m365_originalstarttimezone`|StringType|The start time zone that was set when the event was created. A value of tzone://Microsoft/Custom indicates that a legacy custom time zone was set in desktop Outlook.|
+|`m365_recurrence`|StringType|The recurrence pattern for the event. JSON encoded string, max 4000 in length.for example{\"pattern\":{\"dayOfMonth\":0,\"daysOfWeek\":[\"monday\",\"wednesday\",\"friday\"],\"firstDayOfWeek\":\"sunday\",\"index\":\"first\",\"interval\":1,\"month\":0,\"type\":\"weekly\"},\"range\":{\"startDate\":\"2017-08-14\",\"endDate\":\"2018-08-14\",\"numberOfOccurrences\":0,\"recurrenceTimeZone\":\"Eastern Standard Time\",\"type\":\"endDate\"}}|
+|`m365_reminderminutesbeforestart`|IntegerType|The number of minutes before the event start time that the reminder alert occurs.|
+|`m365_responserequested`|BooleanType|Default is true, which represents the organizer would like an invitee to send a response to the event.|
+|`m365_responsestatus`|StringType|Indicates the type of response sent in response to an event message. JSON encoded string, max 4000 in length.{\"response\": \"String\",\"time\": \"String (timestamp)\"}|
+|`m365_sensitivity`|StringType|Possible values are: normal, personal, private, confidential.|
+|`m365_seriesmasterid`|StringType|The ID for the recurring series master item, if this event is part of a recurring series.|
+|`m365_showas`|StringType|The status to show. Possible values are: free, tentative, busy, oof, workingElsewhere, unknown.|
+|`m365_subject`|StringType|The text of the event's subject line. Primary lookup column|
+|`m365_transactionid`|StringType|A custom identifier specified by a client app for the server to avoid redundant POST operations in case of client retries to create the same event. This is useful when low network connectivity causes the client to time out before receiving a response from the server for the client's prior create-event request. After you set `transactionId` when creating an event, you can't change transactionId in a subsequent update. This property is only returned in a response payload if an app has set it. Optional.|
+|`m365_type`|StringType|The event type. Possible values are: singleInstance, occurrence, exception, seriesMaster. Read-only|
+|`m365_weblink`|StringType|The URL to open the event in Outlook on the web.Outlook on the web opens the event in the browser if you're signed in to your mailbox. Otherwise, Outlook on the web prompts you to sign in. This URL cannot be accessed from within an iFrame.|
+|`m365_grapheventid`|StringType|Unique identifier of the graph event.|
```

Comment:
Change to

| `m365_grapheventId` | guid | Unique identifier of the graph event. |